autoendpoint/src/routers/webpush.rs

```diff
@@ -66,8 +68,33 @@ impl Router for WebPushRouter {
                 }
                 Err(error) => {
                     // We should stop sending notifications to this node for this user
+                    if error.is_timeout() {
+                        self.metrics
+                            .incr_with_tags("error.node.timeout")
+                            .with_tag("node_id", node_id)
+                            .try_send()
+                            .ok();
+                    }
+                    if error.is_connect() {
+                        self.metrics
+                            .incr_with_tags("error.node.connection")
+                            .with_tag("node_id", node_id)
+                            .try_send()
+                            .ok();
+                    }
                     debug!("Error while sending webpush notification: {}", error);
-                    self.remove_node_id(user, node_id.clone()).await?;
+                    if let Some(err) = self.remove_node_id(user, node_id.clone()).await.err() {
+                        match &err.kind {
+                            ApiErrorKind::Database(crate::db::error::DbError::UpdateItem(_)) => {
```

Comment:
I think we can further narrow this to only `UpdateItemError::ConditionalCheckFailed` if we want that?